### DISCUSSION TITLE
how can I remove package?

### DISCUSSION THREAD

**Discussion by Arman-Espiar** [loic-sharma/BaGet#769] [Q&A]
Hi guys, how can I remove package? If I upload a wrong package, how can I correct it?

**Comment by LeaFrock** [comment]
If you wanna delete, see [this](https://loic-sharma.github.io/BaGet/configuration/#enable-package-hard-deletions).
If you wanna overwrite, see [this](https://loic-sharma.github.io/BaGet/configuration/#enable-package-overwrites).

Next time please read the doc site first rather than be a free rider.
